lib/registries/addon/overview/route.ts

```diff
@@ -24,6 +71,10 @@ export default class Overview extends GuidRoute {
         };
     }
 
+    afterModel(this: Overview, model: any) {
+        this.get('setHeadTags').perform(model);
```

Comment:
Could you add a comment explaining why we're not returning this task instance, so future readers know it's intentional?